From ff33f43139268c72eeab79087de965b8bae81b9f Mon Sep 17 00:00:00 2001
Message-ID: <ff33f43139268c72eeab79087de965b8bae81b9f.1780781273.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Sat, 6 Jun 2026 23:26:34 +0200
Subject: [PATCH] Revert "Bug 709438: Support more JBIG2 images."

This reverts commit e458cb36f1d0a99f47e47a25f6e54637f0403e5f.
It requires a jbig2dec which defines JBIG2_OPTIONS_EMBEDDED_FORGIVING.
---
 source/fitz/filter-jbig2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/fitz/filter-jbig2.c b/source/fitz/filter-jbig2.c
index b3233777b..c8612e36f 100644
--- a/source/fitz/filter-jbig2.c
+++ b/source/fitz/filter-jbig2.c
@@ -233,7 +233,7 @@ fz_open_jbig2d(fz_context *ctx, fz_stream *chain, fz_jbig2_globals *globals, int
 
 	options = 0;
 	if (embedded)
-		options |= JBIG2_OPTIONS_EMBEDDED_FORGIVING;
+		options |= JBIG2_OPTIONS_EMBEDDED;
 
 	state->ctx = jbig2_ctx_new((Jbig2Allocator *) &state->alloc, options, globals ? globals->gctx : NULL, error_callback, ctx);
 	if (state->ctx == NULL)
-- 
2.54.0.1146.g083d50a523

